monarch_ingest/pombase/gene_information.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from biolink_model_pydantic.model import Gene
+from koza.cli_runner import koza_app
+
+LOG = logging.getLogger(__name__)
+
+source_name = "pombase_gene_information"
+row = koza_app.get_row(source_name)
+
+# Given that this file is coming from PomBase, I think the biolink model is wrong about all caps
+row["curie"] = row["curie"].replace("PomBase", "POMBASE")
+
+gene = Gene(
+    id=row["curie"],
+    symbol=row["primary gene name"],
+    type=koza_app.translation_table.resolve_term(row["product type"].replace(' ', '_')),
+    source="PomBase",
```

Comment:
@RichardBruskiewich could you comment is this is the correct format for source?  wondering if we need to keep the infores spec in mind here

Reply:
Hi @kshefchek,

you should probably add the **infores:** CURIE prefix to the name, for sure. The identifier is also likely to be all lowercase.  For PomBase, this would look something like `infores:pombase`.  

All that said, I've been a bit out-of-touch with the endpoint of **InfoRes** standards curation. hence I don't know off hand if that would be the "canonical" PomBase InfoRes identifier.  It likely is, given what I know of the rules for identifier formation, but I don't have the public master _InfoRes_ inventory in front of me at this moment.   @mbrush would be able to point you to that master inventory, wherever it is currently being hosted. I'm asking him now (over Slack) but you can also ask him as well.

Reply:
Matt replied. I just emailed you the links to the master spreadsheet of InfoRes. I don't see Pombase in it. Perhaps due to the human biomedical bias of the list (Pombase is more comparative model organisms in Monarch... less visible?). We might find this to be the case for other "primary" knowledge sources (Monarch itself would have an infores as an "aggregating_knowledge_source"). 

BTW, this reminds me.. when serializing records to KGX, from Monarch, we may need to identify how the 'source' field in your record above, gets mapped into the "provenance" of KGX. The PomBase is probably an instance of "primary_knowledge_source" for this purpose (not just 'source' in the KGX).